site/styles.css

```
@@ -0,0 +1,786 @@
+  --text: #e6e6eb;
+  --secondary: #b4b4c0;
+  --muted: #82828e;
+  --primary: #ffb347;
+  --primary-dim: #dc9632;
+  --accent: #67e0e3;
+  --accent-dim: #50b4b7;
+  --success: #76da85;
+  --info: #82aaff;
+  --shadow: 0 34px 96px rgba(0, 0, 0, 0.44);
+  font-family:
+    Inter, ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)